packages/grid/x-data-grid/src/locales/deDE.ts

```diff
@@ -145,12 +145,12 @@ const deDEGrid: Partial<GridLocaleText> = {
   rowReorderingHeaderName: 'Reihen neu ordnen',
 
   // Aggregation
-  // aggregationMenuItemHeader: 'Aggregation',
-  // aggregationFunctionLabelSum: 'sum',
-  // aggregationFunctionLabelAvg: 'avg',
-  // aggregationFunctionLabelMin: 'min',
-  // aggregationFunctionLabelMax: 'max',
-  // aggregationFunctionLabelSize: 'size',
+  aggregationMenuItemHeader: 'Aggregation',
```

Comment:
?

```suggestion
  // aggregationMenuItemHeader: 'Aggregation',
```

Reply:
The spelling for `Aggregation` in German is identical to the English spelling. For reference see [here](https://www.duden.de/rechtschreibung/Aggregation).